packages/evm/lib/nft/MoralisNftProvider.ts

```diff
@@ -0,0 +1,66 @@
+import { ClientTypes } from '@chainify/client';
+import { BaseProvider } from '@ethersproject/providers';
+import { MoralisConfig, NFTAsset, NftTypes } from 'lib/types';
+import Moralis from 'moralis/node';
+import { EvmBaseWalletProvider } from '../wallet/EvmBaseWalletProvider';
+import { EvmNftProvider } from './EvmNftProvider';
+
+export class MoralisNftProvider extends EvmNftProvider {
+    constructor(
+        walletProvider: EvmBaseWalletProvider<BaseProvider>,
+        httpConfig: ClientTypes.AxiosRequestConfig,
+        moralisConfig: MoralisConfig
+    ) {
+        super(walletProvider, httpConfig);
+
+        Moralis.start(moralisConfig).then();
```

Comment:
what is going on here? if `.start()` needs to be awaited, you can't do it like this. You can call `start` on demand when `fetch` is called for example. And after it's started, not call it again